inc/communicationnetworkinventory.class.php

```diff
@@ -490,7 +492,8 @@ function rulepassed($items_id, $itemtype) {
             }
             $inputrulelog['items_id'] = $items_id;
             $inputrulelog['itemtype'] = $itemtype;
-            $inputrulelog['method'] = 'snmpinventory';
+            $inputrulelog['method'] = 'networkinventory';
+            $inputrulelog['criteria'] = $dbu->exportArrayToDB(unserialize($_SESSION['plugin_fusinvsnmp_datacriteria']));
```

Comment:
You should just use `epoxrtArraytoDB` without any DbUtils instanciation

Reply:
Fixed